cdc/log.cc

```diff
@@ -698,7 +698,7 @@ static schema_ptr create_log_schema(const schema& s, const replica::database& db
     }
 
     auto rs = generate_replication_strategy(ksm, db.get_token_metadata().get_topology());
-    auto tombstone_gc_ext = seastar::make_shared<tombstone_gc_extension>(get_default_tombstone_gc_mode(*rs, db.get_token_metadata()));
+    auto tombstone_gc_ext = seastar::make_shared<tombstone_gc_extension>(get_default_tombstone_gc_mode(*rs, db.get_token_metadata(), false));
```

Comment:
Since when do we support cdc and mv table with tablets? What if the  existing table is already using tombstone_gc = repair?

Reply:
2025.4 will be the first version to support MV and CDC for Tablets, before that creating a view or a CDC log table in a tablet keyspace was forbidden.

Reply:
It is good to know . Perhaps we can delay the tablet mv and cdc support a little more until we have the proper repair support.